airflow/models/baseoperator.py

```diff
@@ -1634,7 +1635,7 @@ def map(self, **kwargs) -> "MappedOperator":
 
 
 def _validate_kwarg_names_for_mapping(cls: Type[BaseOperator], func_name: str, value: Dict[str, Any]):
-    if isinstance(str, cls):
+    if isinstance(cls, str):
```

Comment:
Neither the old and new line seem to make sense? Should `cls` be annotated as `Union[Type[BaseOperator], str]` instead?

Reply:
Yes, it should be -- I corrected that type annotation on the MappedOperator but not here.

Reply:
I pushed a commit to fix this (and a few other annotations)